bigquery/tests/unit/test_client.py

```diff
@@ -3111,54 +3111,6 @@ def test_list_rows_errors(self):
         with self.assertRaises(TypeError):
             client.list_rows(1)
 
-    def test_list_partitions(self):
```

Comment:
You're probably going to need some sort of unit test for 100% unit test coverage.

Reply:
P.S. I agree that a system test is more useful, but they are quite slow to run. The purpose of the 100% coverage by unit tests is so that we can be mostly confident things won't break while developing by just running unit tests.